cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -19,6 +19,9 @@ spec:
         version: v11
         kubernetes.io/cluster-service: "true"
     spec:
+{% if grains['cloud'] is defined and grains['cloud'] in [ 'vsphere', 'photon-controller' ] %}
```

Comment:
Re hostnetwork: why is the kubedns pod not able to reach the actual DNS server? 

> It was slow because the pod couldn't access the parent DNS because it didn't have connectivity to the node's network.

I don't understand this explanation. So the pod can't ping another node's ip? kube-dns needs to talk to the apiserver anyway, how does that work?